app/styles/globals.scss

```diff
@@ -18,6 +18,14 @@
   -webkit-tap-highlight-color: transparent;
 }
 
+html {
+  font-size: 14px;
```

Comment:
I feel like going this way, the smallest text i.e. dose description rows are just a tad too small? The current size seems perfect. Maybe for fonts we want to be a bit more selective?

Reply:
Yeah, fair - happy to be more considered :)